File: nan21.dnet.module.fi/nan21.dnet.module.fi.ui.extjs/src/main/webapp/src/ds/AssetCategoryAcctDs.js
/*
 * DNet eBusiness Suite
 * Copyright: 2008-2012 Nan21 Electronics SRL. All rights reserved.
 * Use is subject to license terms. 
 */   	 
Ext.define("net.nan21.dnet.module.fi.asset.ds.model.AssetCategoryAcctDs" ,{
	extend: 'Ext.data.Model',
	fields: [ 
		{name:"id", type:"int", useNull:true},
		{name:"uuid", type:"string"},
		{name:"clientId", type:"int", useNull:true},
		{name:"createdAt", type:"date", dateFormat:Dnet.MODEL_DATE_FORMAT},
		{name:"modifiedAt", type:"date", dateFormat:Dnet.MODEL_DATE_FORMAT},
		{name:"createdBy", type:"string"},
		{name:"modifiedBy", type:"string"},
		{name:"version", type:"int", useNull:true},
		{name:"entityFQN", type:"string"},
		{name:"assetCategoryId", type:"int", useNull:true},
		{name:"assetCategory", type:"string"},
		{name:"accSchemaId", type:"int", useNull:true},
		{name:"accSchema", type:"string"},
		{name:"deprecAccountId", type:"int", useNull:true},
		{name:"deprecAccount", type:"string"},
		{name:"acmlDeprecAccountId", type:"int", useNull:true},
		{name:"acmlDeprecAccount", type:"string"}
	]
});

Ext.define("net.nan21.dnet.module.fi.asset.ds.model.AssetCategoryAcctDsFilter" ,{
	extend: 'Ext.data.Model',
	fields: [ 
		{name:"id", type:"int", useNull:true},
		{name:"uuid", type:"string"},
		{name:"createdAt", type:"date", dateFormat:Dnet.MODEL_DATE_FORMAT},
		{name:"createdAt_From",type:"date", dateFormat:Dnet.MODEL_DATE_FORMAT},
		{name:"createdAt_To",type:"date", dateFormat:Dnet.MODEL_DATE_FORMAT},
		{name:"modifiedAt", type:"date", dateFormat:Dnet.MODEL_DATE_FORMAT},
		{name:"modifiedAt_From",type:"date", dateFormat:Dnet.MODEL_DATE_FORMAT},
		{name:"modifiedAt_To",type:"date", dateFormat:Dnet.MODEL_DATE_FORMAT},
		{name:"createdBy", type:"string"},
		{name:"modifiedBy", type:"string"},
		{name:"entityFQN", type:"string"},
		{name:"assetCategoryId", type:"int", useNull:true},
		{name:"assetCategoryId_From",type:"int", useNull:true},
		{name:"assetCategoryId_To",type:"int", useNull:true},
		{name:"assetCategory", type:"string"},
		{name:"accSchemaId", type:"int", useNull:true},
		{name:"accSchemaId_From",type:"int", useNull:true},
		{name:"accSchemaId_To",type:"int", useNull:true},
		{name:"accSchema", type:"string"},
		{name:"deprecAccountId", type:"int", useNull:true},
		{name:"deprecAccountId_From",type:"int", useNull:true},
		{name:"deprecAccountId_To",type:"int", useNull:true},
		{name:"deprecAccount", type:"string"},
		{name:"acmlDeprecAccountId", type:"int", useNull:true},
		{name:"acmlDeprecAccountId_From",type:"int", useNull:true},
		{name:"acmlDeprecAccountId_To",type:"int", useNull:true},
		{name:"acmlDeprecAccount", type:"string"}
	] 
});
	
Ext.define("net.nan21.dnet.module.fi.asset.ds.param.AssetCategoryAcctDsParam", {
	extend: 'Ext.data.Model',
	fields: [ 
	]
}); 
 

File: nan21.dnet.module.fi/nan21.dnet.module.fi.presenter/src/main/java/net/nan21/dnet/module/fi/asset/ds/model/AssetCategoryAcctDs.java
/*
 * DNet eBusiness Suite
 * Copyright: 2008-2012 Nan21 Electronics SRL. All rights reserved.
 * Use is subject to license terms. 
 */
package net.nan21.dnet.module.fi.asset.ds.model;

import net.nan21.dnet.core.presenter.model.base.AbstractAuditableDs;

import net.nan21.dnet.module.fi.asset.domain.entity.AssetCategoryAcct;
import net.nan21.dnet.core.api.annotation.Ds;
import net.nan21.dnet.core.api.annotation.DsField;

@Ds(entity = AssetCategoryAcct.class)
public class AssetCategoryAcctDs extends AbstractAuditableDs<AssetCategoryAcct> {

    public static final String fASSETCATEGORYID = "assetCategoryId";
    public static final String fASSETCATEGORY = "assetCategory";
    public static final String fACCSCHEMAID = "accSchemaId";
    public static final String fACCSCHEMA = "accSchema";
    public static final String fDEPRECACCOUNTID = "deprecAccountId";
    public static final String fDEPRECACCOUNT = "deprecAccount";
    public static final String fACMLDEPRECACCOUNTID = "acmlDeprecAccountId";
    public static final String fACMLDEPRECACCOUNT = "acmlDeprecAccount";

    @DsField(join = "left", path = "assetCategory.id")
    private Long assetCategoryId;

    @DsField(join = "left", path = "assetCategory.code")
    private String assetCategory;

    @DsField(join = "left", path = "accSchema.id")
    private Long accSchemaId;

    @DsField(join = "left", path = "accSchema.code")
    private String accSchema;

    @DsField(join = "left", path = "deprecAccount.id")
    private Long deprecAccountId;

    @DsField(join = "left", path = "deprecAccount.code")
    private String deprecAccount;

    @DsField(join = "left", path = "acmlDeprecAccount.id")
    private Long acmlDeprecAccountId;

    @DsField(join = "left", path = "acmlDeprecAccount.code")
    private String acmlDeprecAccount;

    public AssetCategoryAcctDs() {
        super();
    }

    public AssetCategoryAcctDs(AssetCategoryAcct e) {
        super(e);
    }

    public Long getAssetCategoryId() {
        return this.assetCategoryId;
    }

    public void setAssetCategoryId(Long assetCategoryId) {
        this.assetCategoryId = assetCategoryId;
    }

    public String getAssetCategory() {
        return this.assetCategory;
    }

    public void setAssetCategory(String assetCategory) {
        this.assetCategory = assetCategory;
    }

    public Long getAccSchemaId() {
        return this.accSchemaId;
    }

    public void setAccSchemaId(Long accSchemaId) {
        this.accSchemaId = accSchemaId;
    }

    public String getAccSchema() {
        return this.accSchema;
    }

    public void setAccSchema(String accSchema) {
        this.accSchema = accSchema;
    }

    public Long getDeprecAccountId() {
        return this.deprecAccountId;
    }

    public void setDeprecAccountId(Long deprecAccountId) {
        this.deprecAccountId = deprecAccountId;
    }

    public String getDeprecAccount() {
        return this.deprecAccount;
    }

    public void setDeprecAccount(String deprecAccount) {
        this.deprecAccount = deprecAccount;
    }

    public Long getAcmlDeprecAccountId() {
        return this.acmlDeprecAccountId;
    }

    public void setAcmlDeprecAccountId(Long acmlDeprecAccountId) {
        this.acmlDeprecAccountId = acmlDeprecAccountId;
    }

    public String getAcmlDeprecAccount() {
        return this.acmlDeprecAccount;
    }

    public void setAcmlDeprecAccount(String acmlDeprecAccount) {
        this.acmlDeprecAccount = acmlDeprecAccount;
    }

}
